src/planscape/planning/serializers.py

```diff
@@ -6,14 +6,15 @@
 from planning.models import (PlanningArea, Scenario, ScenarioResult)
 
 # TODO: flesh all serializers more for better maintainability.
-
 class PlanningAreaSerializer(gis_serializers.GeoFeatureModelSerializer):
     scenario_count = IntegerField(read_only=True, required=False)
-    scenario_latest_updated_at = DateTimeField(read_only=True, required=False)
+    latest_updated = serializers.SerializerMethodField()
```

Comment:
Since this replaces some of the metadata fields for a planning area, would you mind updating the documenting comments under list_planning_areas and get_planning_area_by_id that reference scenario_latest_updated_at in planning/views?